app/views/browse/build-config.html

```diff
@@ -386,28 +386,28 @@ <h3>Triggers <a href="{{'build-triggers' | helpLink}}" target="_blank"><span cla
                               <dt>Bitbucket Webhook URL:
                               </dt>
                               <dd>
-                                <copy-to-clipboard clipboard-text="buildConfig.metadata.name | webhookURL : trigger.type : trigger.bitbucket.secret : project.metadata.name"></copy-to-clipboard>
+                                <copy-webhook-url build-config-name="buildConfig.metadata.name" trigger-type="trigger.type" project-name="project.metadata.name" secret="trigger.bitbucket" webhook-secrets="webhookSecrets"></copy-webhook-url>
                               </dd>
                           </div>
                           <div ng-switch-when="GitHub">
                               <dt>GitHub Webhook URL:
                               </dt>
-                              <dd>
-                                <copy-to-clipboard clipboard-text="buildConfig.metadata.name | webhookURL : trigger.type : trigger.github.secret : project.metadata.name"></copy-to-clipboard>
+                              <dd class="with-warning">
```

Comment:
I don't see where `with-warning` styles are added. Why only for GitHub webhooks?